website/layouts/index.html

```diff
@@ -72,7 +72,9 @@ <h2>Recently Published</h2>
           <h2>Collaboration is Welcomed!</h2>
           <p><img class="icon" src="/images/collaboration.svg"/></p>
           <p>We want to invite everyone to help to guide the future of Enterprise Contract.</p>
-          <p>Find our <a href="https://github.com/enterprise-contract/" title="View the Enterprise Contract organization on GitHub">code on GitHub</a>.</p>
+          <p>View the <a href="https://github.com/enterprise-contract/ec-cli" title="View the Enterprise Contract organization on GitHub">code on GitHub</a></p>
+          <p>Found a bug with enterprise contract? Open an <a href="">issue</a></p>
+          <p>Have questions? Enhancement ideas? Join the <a href="https://github.com/enterprise-contract/ec-cli/discussions">discussion</a>!</p>
```

Comment:
What if it's not ec-cli related?

Reply:
Yeah, I think this might be the biggest factor for creating a separate page. I'll follow up with more.